example-get-started/code/src/evaluate.py

```diff
@@ -100,13 +100,14 @@ def main():
         test, _ = pickle.load(fd)
 
     # Evaluate train and test datasets.
-    live = Live(EVAL_PATH)
-    evaluate(model, train, "train", live, save_path=EVAL_PATH)
-    evaluate(model, test, "test", live, save_path=EVAL_PATH)
-    live.make_summary()
+    with Live(EVAL_PATH, cache_images=True, dvcyaml=False) as live:
+        evaluate(model, train, "train", live, save_path=EVAL_PATH)
+        evaluate(model, test, "test", live, save_path=EVAL_PATH)
 
-    # Dump feature importance plot.
-    save_importance_plot(live, model, feature_names)
+        live.make_summary()
```

Comment:
if we do context manager then we don't need to explicitly call this
```suggestion
```

Reply:
I think it was intentional because we don't want to make dvc.yaml, report, etc. It is a bit advanced but we will either have to do it this way or enable some options like `with Live(dvcyaml=False)`.

Reply:
Then we need to add `report=None` on top of this because we would be now generating the report

Reply:
You mean if we use the context manager, we need `with Live(dvcyaml=False, report=None)`?